manifests/role.yaml

```diff
@@ -0,0 +1,21 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: peribolos-role
+  namespace: peribolos
+rules:
+- apiGroups:
+  - "tekton.dev"
+  resources:
+  - "taskRuns"
```

Comment:
I think according to the specification this should reference the plural name of the resource

```suggestion
  - "taskruns"
```

![image](https://user-images.githubusercontent.com/7453394/162071672-30c00d3c-a36a-4152-bc24-0f10b7f34dd0.png)

Reply:
Yes it should thanks!